api/src/main/java/io/strimzi/api/kafka/model/KafkaExporterSpec.java

```diff
@@ -77,6 +80,28 @@ public void setTopicRegex(String topicRegex) {
         this.topicRegex = topicRegex;
     }
 
+    @Description("Regular expression to specify which consumer groups to exclude. " +
+            "Default value is `^$`.")
```

Comment:
if you are going to get the Jakub's suggestion (which I agree on), remember to update the documentation here as well.